BuoyTest/src/components/cocktails/details/CocktailView.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Stylesheet, View, Text, Image, ScrollView } from 'react-native';
+import styles from './styles';
+
+class CocktailView extends Component {
+  state = {}
+  render() {
+    const { cocktail } = this.props;
+    if (cocktail != null) {
+      let arrayIngredients = []
+      for (let index = 1; index <= 15; index++) {
+          const keyIngredient = "strIngredient"+index
+          const keyMeasure = "strMeasure"+index
+          if (cocktail[keyIngredient] && cocktail[keyMeasure]) {
+              arrayIngredients.push(<Text key={index} style={styles.ingredient}>{cocktail[keyMeasure]+" "+cocktail[keyIngredient]}</Text>)
```

Comment:
For string concatenation we use

``${cocktail[keyMeasure]} ${cocktail[keyIngredient]}``

Reply:
Updated!